db/active_replicator_checkpointer.go

```diff
@@ -37,30 +41,47 @@ type Checkpointer struct {
 	// lastCheckpointSeq is the last checkpointed sequence
 	lastCheckpointSeq string
 
+	// SGR1 Checkpoint Migration:
+	// sgr1CheckpointID can be set to be used as a fallback checkpoint if none are found
+	sgr1CheckpointID string
+	// sgr1CheckpointOnRemote determines which end of the replication to fetch the SGR1 checkpoint from.
+	// For SGR1 pull replications, the checkpoints are stored on the source, for push they're stored on the target.
+	sgr1CheckpointOnRemote bool
+	// InsecureSkipVerify skips TLS verification when fetching a remote SGR1 checkpoint.
+	sgr1RemoteInsecureSkipVerify bool
+	// passiveDBURL is used to fetch local SGR1 checkpoints.
+	sgr1LocalPassiveDBURL *url.URL
```

Comment:
I think this is just used to fetch remote SGR1 checkpoints, isn't it?  

I wonder if we should just rename ActiveReplicatorConfig.PassiveDBURL to ActiveReplicatorConfig.RemoteDBURL, and just store it here as remoteDBURL (since it isn't really something sgr1-specific).